packages/react-components/react-table/src/components/TableCellLayout/TableCellLayout.test.tsx

```diff
@@ -2,24 +2,25 @@ import * as React from 'react';
 import { render } from '@testing-library/react';
 import { TableCellLayout } from './TableCellLayout';
 import { isConformant } from '../../common/isConformant';
+import { TableCellLayoutProps } from './TableCellLayout.types';
 
 describe('TableCellLayout', () => {
   isConformant({
```

Comment:
```suggestion
  isConformant<TableCellLayoutProps>({
```

This should be enough instead of the casting